PyDSS/pyControllers/Controllers/PvController.py

```diff
@@ -70,14 +72,37 @@ def __init__(self, PvObj, Settings, dssInstance, ElmObjectList, dssSolver):
 
         self.update = [self.ControlDict[Settings['Control' + str(i)]] for i in [1, 2, 3]]
 
+        if Settings["Priority"] == "Var":
+            PvObj.SetParameter('WattPriority',"False")
+        else:
+            PvObj.SetParameter('WattPriority', "True")
+
         #self.QlimPU = self.__Qrated / self.__Srated if self.__Qrated < self.__Srated else 1
         self.QlimPU = min(self.__Qrated / self.__Srated, Settings['QlimPU'], 1.0)
+        print("self.QlimPU:  ", self.QlimPU)
```

Comment:
Can you delete this print statement? I assume it's a stale debug print.

Reply:
fixed